src/cmd-buildextend-live

```diff
@@ -415,7 +415,12 @@ def generate_iso():
         # can be EFI/fedora or EFI/redhat
         grubpath = glob.glob(os.path.join(tmpisoroot, 'EFI/*/grub.cfg'))
         shutil.move(grubpath[0], os.path.join(tmpisoroot, 'boot/grub/grub.cfg'))
-
+        for key in list(files_with_karg_embed_areas):
+            if re.search(r"/EFI/.*/grub.cfg", key):
+                PATH_ = os.path.join(tmpisoroot, 'boot/grub/grub.cfg'))
```

Comment:
You can declare this before line 417, then you'd be able to use it in the `shutil.move()` call too. Also, let's use a clearer variable name, e.g. `boot_grubpath`.

Reply:
hmm ... I got it. I will do it

Reply:
https://github.com/coreos/coreos-assembler/pull/2831/commits/4bc93db282e2e4c45faa8d6919ab2e26b10a0a90